### ISSUE TITLE
Documentation Updates

### ISSUE THREAD

**Issue by schmidtbag** [aarch64-laptops/debian-cdimage#13]
I've been trying to get Debian installed on my Lenovo C630, but after the installation completes and the computer reboots, it doesn't go any farther than GRUB, regardless of whether I follow the DtbLoader instructions (of which, I'm not confident I did right - more on that later).  There are some parts of the documentation I'm a little confused by:

- Under "Installation Tips", I think it would be handy to point out that using dd is a fast effective way to get the ISO to boot from a flash drive.
- Also under "Installation Tips", it isn't clear how partitioning _should_ be done.  For me, I want to boot off an external drive (for now).  I found you need a fat16 boot+esp partition; the Guided option in the installer will do this for you.
- Under "AArch64 Quirks", it isn't entirely obvious that you have to boot into the _installer's_ UEFI shell.  Not a big deal, just a little unclear.
- I don't get what I'm doing for steps 3 and 4 in the quirks section.  To my understanding, this is what you would need to do if you're dual-booting Windows while installing to the internal SSD, but I can't tell entirely if that's right because I don't meddle with this sort of stuff.  I was already able to boot from my external drive without following these steps, but I'm not sure if there's more to it.  I say that because whether I followed those steps or not, I never succeeded in getting the laptop to do anything beyond the GRUB menu - telling it to boot Debian just sat there and did nothing.  No errors either.  I'm sure I'm doing something wrong, but I don't know what it is.

TL;DR: Some adjustments to this guide for booting on an external drive would help solve some of these quirks.  I feel like I'm half way there but I don't know what to do when I'm not given any errors.

**Comment by shawnguo2** [comment]
The document is really meant for installing Debian onto internal UFS disk rather than external drive.
